modules/baseline-router-performance.adoc

```diff
@@ -1,12 +1,9 @@
 [id="baseline-router-performance_{context}"]
-= Baseline router performance
+= Baseline Ingress Controller (router) performance
 
-The {product-title} router is the Ingress point for all external traffic
-destined for {product-title} services.
+The {product-title} Ingress Controller, or router, is the Ingress point for all external traffic destined for {product-title} services.
```

Comment:
Should we _always_ capitalize "Ingress"? With "Ingress point", it seems like a common/general usage and I would think of lowercasing it.

Reply:
Yeah...since it's a Kubernetes resource and they capitalize it, I've been following that pattern. It's definitely something we can look into in the future (especially since it looks like they're now not capitalizing controller in Ingress controller).